### ISSUE TITLE
White screen occasionally while reading epub files

### ISSUE THREAD

**Issue by zhliu0124** [koreader/koreader#3287]
* KOReader version: v2015.11-1270 and koreader-nightly-20170821
* Device: kpw3

#### Issue
The screen turns completely white while reading epub files, usually once every n minutes (sorry I didn't time it). The screen is still responsive when it's white. Touching the screen would trigger the behavior it is supposed to do (for example, touching the screen in the center would turn to the next page and bring the text back)

It does it all by itself and to the whole screen. It came in the middle of my reading, not triggered by touching.

**Comment by frankyifei** [comment]
maybe it's because the books' css style, css page-break is not well supported. try apply another style or remove style in the menu

**Comment by Frenzie** [comment]
By white screen do you mean:

1. It does it all by itself and to the whole screen.
2. Or, as I think @frankyifei interpreted, an occasional extra white page.

**Comment by poire-z** [comment]
It sounds like it's this epub.css thing I mentionned in #2394 (last item, that didn't make it). I still get that blank page when next/prev chapter on 20% of the epub i read.

**Comment by zhliu0124** [comment]
Reply to @Frenzie:

> By white screen do you mean:
> It does it all by itself and to the whole screen.
> Or, as I think @frankyifei interpreted, an occasional extra white page.

It does it all by itself and to the whole screen. It came in the middle of my reading, not triggered by touching. It seems not as @frankyifei suggested.

**Comment by Frenzie** [comment]
Thanks. Just making sure. I don't think there's anything in KOReader that would cause that. It might be something from the Kindle system, but I don't really know anything about Kindles.

**Comment by robert00s** [comment]
I have the same issue. It came not triggering by touching. No more than once in session. Mostly at the beginning of reading. I'm pretty sure that it's not a KO problem but something in Kindle system (maybe screensaver, changing special offers I don't know).

@zhliu0124 When you have white page try diagonal swipe. You don't need to turn page to refresh screen.

**Comment by zhliu0124** [comment]
@robert00s Thank you!

**Comment by robert00s** [comment]
I noticed that this happens when the wifi connection is lost and KO is open.
@zhliu0124 Can you check that. Maybe try read some time with wifi disabled. Than problem shouldn't occur.
I don't have idea how to fix it. Do KO perform any action when the network connection is lost?

**Comment by Frenzie** [comment]
No, not until #3078 is finished afaik.

**Comment by zhliu0124** [comment]
@robert00s 
Thanks! This still happens when my kindle with airplane mode on, so I don't think this is the reason.

**Comment by robert00s** [comment]
@zhliu0124 
Do you have still that problem after upgrade KPVBooklet like in #3363?

**Comment by zhliu0124** [comment]
@robert00s 
Yes, unfortunately. I was hoping it could be solved by updating the KPVBooklet plugin.

**Comment by focus-cst** [comment]
Same here with fb2 files.
And i think this is related to suspended system UI etc, because as far as i can see, it never happens when i open fb2 first time using booklet from kindle UI after book reboot, nor it happens if i switch book from inside koreader.
But, if/when i exit koreader and open another book from kindle UI, i have whie screen by itself time to time (and also sometimes battery indicator popup from system UI).

**Comment by NiLuJe** [comment]
Possibly fixed w/ #3955

Reopen if need be ;).

**Comment by robert00s** [comment]
Problem still exist. White screen show after 5-10 sec after open book. This happens always when wifi is on. I'm attaching short movie.
[movie.zip](https://github.com/koreader/koreader/files/2195932/movie.zip)

**Comment by NiLuJe** [comment]
Huh.

Is that a Special Offers device? Does it happen without the smart cover?

**Comment by NiLuJe** [comment]
Note that the full black flash before that is also spurious, and shouldn't happen.

**Comment by NiLuJe** [comment]
Nope, not a smart cover thing either, can't reproduce with a magnet, it behaves as expected (i.e., exactly like a power button press).

**Comment by NiLuJe** [comment]
Can't test w/ WiFi on, unfortunately (I'm not on the latest FW).

**Comment by robert00s** [comment]
I have device without SO. Firmware version 5.8.2. I use KPVBooklet v0.6.4.
This happen also without using smart cover.
This first full black flash happen for a very long time > 1 year.
I have screensaver hack. I'll try to disable it and let you know if it helped.

**Comment by NiLuJe** [comment]
It won't ;). Starting from KUAL might, though, and probably explains why I
can't reproduce it.

**Comment by NiLuJe** [comment]
Yeah, got another confirmation in #4413 that this was KPVBooklet specific.

I have no idea if it's even possible to work-it around.

**Comment by NiLuJe** [comment]
Closing in favor of #4605 to make searches more accurate.
